library/Freckle/App.hs

```diff
@@ -248,8 +248,14 @@ instance PrimMonad m => PrimMonad (AppT app m) where
 instance (Monad m, HasTracer app) => MonadTracer (AppT app m) where
   getTracer = view tracerL
 
-instance Applicative m => XRay.MonadTracer (AppT app m) where
-  getVaultData = pure Nothing
+-- instance Applicative m => HasTracer (AppT app m) where
+--   getVaultData = pure Nothing
```

Comment:
I would revert this. The `XRay.MonadTracer` instance still works (and is needed) for things opted into `XRay`; they just can't use `MonadSqlTx` yet. And the commented `HasTracer` instance isn't valid even if uncommented.

Reply:
oof, I've further confused this because this isn't even something I meant to remove.